fr/releases/centreon-commercial-extensions.md

```diff
@@ -130,6 +130,13 @@ Référez-vous à la [documentation dédiée](../graph-views/install.html#centre
 
 ## Centreon Auto Discovery
 
+### 20.10.3
+
```

Comment:
Missing mention to those fixes : 
[HD/Provider/Install/Upgrade] Limit provider definition version that is imported to 2.0 in 20.10/20.04
[Core] Creation forms generate 400 errors